macros/core/PGauxiliaryFunctions.pl

```diff
@@ -266,57 +268,59 @@ sub gcf {
 	return gcf($b, @_);
 }
 
-=head3 gcd function 
+=head3 gcd function
 
 =pod
 
 	Usage: gcd(@arr);
 
-returns the greatest common divisor of the array @arr of integers.  
+returns the greatest common divisor of the array @arr of integers.
 
 Example:
+
 	gcd(20,30,45) returns 5.
 
-Note: this is just an alias for gcf. 
+Note: this is just an alias for gcf.
 
 =cut
 
 sub gcd {
 	return gcf(@_);
 }
 
-=head3 random_coprime function 
+=head3 random_coprime function
 
 =pod
 
 	Usage: random_coprime(array of array_refs);
 
-returns relatively prime integers. The arguments should be references to arrays of integers.  This returns an n-tuple of relatively prime integers, 
-each one coming from the corresponding array. Random selection is uniform among all possible tuples that are relatively prime.
-This does not consider (0,0) to be relatively prime.
+returns relatively prime integers.  The arguments should be references to arrays of integers.  This returns an
+n-tuple of relatively prime integers, each one coming from the corresponding array. Random selection is uniform
+among all possible tuples that are relatively prime.  This does not consider (0,0) to be relatively prime.
 
-This function may return an n-tuple where pairs are not coprime.  This returns n-tuples where the largest (in absolute) common factor is 1. 
+This function may return an n-tuple where pairs are not coprime.  This returns n-tuples where the largest
+(in absolute) common factor is 1.
 
 In array context, returns an array. Otherwise, an array ref.
 
-Examples: 
+Examples:
 
 	random_coprime([1..9],[1..9]) may return (2,9) or (1,1) but not (6,8)
 	random_coprime([-9..-1,1..9],[1..9],[1..9]) may return (-3,7,4), (-1,1,1), or (-2,2,3) but not (-2,2,4)
 
-Note: in the example above (-2,2,3) is valid because not all three share a factor greater than 1.  If you don't want to 
-allow pairs of numbers to be coprime, see random_pairwise_coprime. 
+Note: in the example above (-2,2,3) is valid because not all three share a factor greater than 1.
+If you don't want to allow pairs of numbers to be coprime, see random_pairwise_coprime.
```

Comment:
This is an old mistake, but i saw it since it's in the diff. I think this line should be:
```suggestion
If you don't want to allow pairs of numbers to have a common factor, see random_pairwise_coprime.
```
If there is no more to edit in this PR, you could accept my code suggestion here to fix this.

Reply:
thanks, I'll add a small fix.